libs/ui-lib/lib/ocm/components/clusterConfiguration/platformIntegration/constants.tsx

```diff
@@ -0,0 +1,19 @@
+import { NUTANIX_CONFIG_LINK, VSPHERE_CONFIG_LINK } from '../../../../common';
+import { PlatformType } from '../../../api';
+
+export const ExternalPlatformLabels: { [key in PlatformType]: string } = {
```

Comment:
I like that you extracted this data into a `constants` module.

Due to that, now the data in `ClusterPlatformIntegrationHint` should be unnecessary, and since the component is not used anywhere, it should be deleted entirely.

Usages of `integrationPlatformLinks` should refer to this new file.